sdks/python/apache_beam/transforms/environments_test.py

```diff
@@ -53,10 +63,14 @@ def test_environment_encoding(self):
                             state_cache_size=0, data_buffer_time_limit_ms=0),
                         SubprocessSDKEnvironment(command_string=u'foö')):
       context = pipeline_context.PipelineContext()
-      self.assertEqual(
-          environment,
-          Environment.from_runner_api(
-              environment.to_runner_api(context), context))
+      proto = environment.to_runner_api(context)
+      reconstructed = Environment.from_runner_api(proto, context)
+      self.assertEqual(environment, reconstructed)
+      self.assertEqual(proto, reconstructed.to_runner_api(context))
+
+  def test_sdk_capabilities(self):
+    sdk_capabilities = environments.python_sdk_capabilities()
+    self.assertIn(common_urns.coders.LENGTH_PREFIX.urn, sdk_capabilities)
```

Comment:
Probably test for every expected capability here ? And also force failure if the test has not been updated to reflect a new capability ?

The test can be updated as the list of capabilities are updated.